utilities/pipelines/sharedScripts/Get-ModuleTestFileList.ps1

```diff
@@ -33,7 +33,7 @@ function Get-ModuleTestFileList {
         [string] $ModulePath,
 
         [Parameter(Mandatory = $false)]
-        [string] $SearchFolder = '.test',
+        [string] $SearchFolder = 'tests',
```

Comment:
I guess we can remove line 45 😄 should we di it right here?

Reply:
removed